text/0000-channel-participation-api-without-system-channel.md

```diff
@@ -0,0 +1,536 @@
+
+The _local orderer participation admin_  role will have the same authentication and authorization
+as the _operations admin_ (i.e. the same mutual TLS certificate). We choose this option because it is the simplest 
+to implement at this time. This is expected to change in the future (see below).
+
+## Participation management
+We intend to implement the channel participation management functionality in a new package 
+(`orderer/common/channelparticipation`). In this package a new class will keep a map of channel names and their status,
+and will be in charge of coordinating the operations of following or joining a channel as a member. 
+Every channel that needs to follow or catch-up on the cluster will have a dedicated goroutine. When the channel reaches
+a state where it can join the cluster, the respective methods in `orderer/common/multichannel/registrar.go` will be 
+invoked in order to start the chain runtime.
+
+When a channel is first joined with a config block, the channel's ledger folder is created, and inside it the config 
+block is saved in a file called `.join-config-block`. The existence of this file is a signal that the join process is 
+active, and should be resumed after restart, even if the orderer crashes or is terminated. 
+This block will be used as a "bootstrap" for pulling blocks from other OSNs, and its height 
+an indication for when catch-up is complete. If the OSN is a member of the cluster when catch-up
+is complete, the node will start the chain, otherwise it will continue to follow and check 
+incoming config blocks. When the node joins the cluster and starts the respective chain run-time component,
+it will delete the `.join-config-block` file.
+
+The process of a join includes fetching blocks from other OSNs. For that purpose we will reuse code from 
+`orderer/common/cluster`, in particular the method that pulls a single named channel 
+`Replicator.PullChannel(channel string)`.
+
+Deleting a channel is achieved by first adding to the ledger folder a file called `.remove`, again, to mark it for 
+removal across restarts. Then, the runtime components of the channel are terminated (using `.../multichannel`),
+and finally, if respective flag is set, the folder is deleted.
+
+Listing the channel or channel is achieved by querying the channel map in the `.../channelparticipation` package 
+implementation.
+
+The interface of executing channel participation commands is expected to stay stable even if the  authentication and 
+authorization mechanisms change, or even if this API is separated on to a different service endpoint.
+
+## Boot sequence of Orderer
+In the boot sequence of the orderer (`orderer/server/main.go`) we will have to check whether the orderer is working
+with or without the system channel, and then start either the current on-boarding code (eventually calling 
+`orderer/common/cluster/Replicator`) or the new channel participation component.
```

Comment:
I was initially thrown for a loop here, but I think you mean the component which implements the following/joining go routines, not the the participation service which still allows for the deletion of the system channel and listing of channels.  Could be clarified.

Reply:
I clarified the text.